components/InstalmentsInfo.js

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+
+const InstalmentsInfo = () => (
+  <div className="infoCards _mt-6 _shadow-md">
+    <div className="_p-5 _bg-gray-100  _rounded">
+      <h3 className="_w-9/12 _text-left _text-lg _text-blue-100 _font-semibold _font-primary _mb-2 ">
+        Betaal bijvoorbeeld in 3, 6 of 24 termijnen
+      </h3>
+      <div className="_flex">
+        <p className="_text-sm _text-gray-300 _font-secondary _leading-snug">
+          In Mijn wehkamp kun je zelf de looptijd en daarmee de kosten van je
+          krediet instellen. Heb je geen openstaand saldo of
+          betaalverplichtingen meer, dan kun je je kredietovereenkomst zelf
+          beëindigen in Mijn wehkamp. Je kredietovereenkomst wordt dan binnen 2
+          dagen afgemeld bij BKR.
+        </p>
+        <img src="static/svg/betaal_bijvoorbeeld.svg" className="_w-0 sm:_w-32"/>
```

Comment:
```suggestion
        <img src="static/svg/betaal_bijvoorbeeld.svg" className="hidden sm:inline-block sm:_w-32" alt="" />
```

also, the image is different in the design? (a clock in Github, a stack of coins in the design)

Reply:
Checked in with Cristina, the design  was updated with the clock svg